File: week-03/day-02/write-single-line.ts
// Open a file called 'my-file.txt'
// Write your name in it as a single line
// If the program is unable to write the file,
// then it should print an error message like: 'Unable to write file: my-file.txt'

'use strict';
declare function require(path: string): any;
export { };

const fs = require('fs');
const charEncoding = 'utf-8';

function readFromFile(fileName: string): string {
  try {
    return fs.readFileSync(fileName, charEncoding);
  } catch (e) {
    console.log(e.message);
    return null;
  }
}

function writeToAFile(fileName: string, data: any): void {
  fs.appendFileSync(fileName, '\n'+data);
}
let data: string = 'This is the 2nd sentence what i would like to put there';
let fileName: string = 'my-file.txt';
writeToAFile(fileName,data);

File: week-06/day-04/writesingleline.c
#include <stdio.h>
#include <string.h>

// Open a file called "my-file.txt"
// Write your name in it as a single line

int main()
{
    char filename;
    FILE *fptr;
    char line[1000];
    if ((fptr = fopen(filename, "r")) == NULL)
    {
        printf("Error! opening file");
        // Program exits if file pointer returns NULL.
        exit(1);
    }

    // reads text until newline
    int counter = 0; /*Number of lines*/

    return 0;
}